### PR TITLE
MicroBit Shake Event

### PR DESCRIPTION
**Pull request by epeach**
Jira Ticket: https://codedotorg.atlassian.net/browse/STAR-1059

Adds the shake board event for the MicroBit

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/lib/kits/maker/commands.js [hunk 1]
```diff
@@ -106,7 +107,8 @@ export function boardConnected() {
  */
 export function onBoardEvent(opts) {
   let {component, event, callback} = opts;
-  if (BOARD_EVENT_ALIASES[event]) {
+  // Look up aliases for CircuitPlaygroundBoard
+  if (BOARD_EVENT_ALIASES[event] && !(board instanceof MicroBitBoard)) {
```
Comment (maddiedierker) [inline apps/src/lib/kits/maker/commands.js:111]: why don't we want to do this lookup for micro:bit boards?

Comment (epeach) [reply apps/src/lib/kits/maker/commands.js:111]: We surface the 'shake' event to the student, however, the CircuitPlayground doesn't recognize that event, so we alias it to singleTap event that the board recognizes. MicroBit handles the shake event (and not the tap events) so we don't need to alias it. The FakeBoard is just a CP FakeBoard, so we need to alias for the CP and FakeCP, but not the MicroBit.

Comment (maddiedierker) [reply apps/src/lib/kits/maker/commands.js:111]: ohhh, makes sense. thank you!

**Review by maddiedierker — APPROVED** [review]
i was holding out for a "bmo shake" gif, but this is the best i could find:

bmo-dance

**Pull request merged**